cpp/ps0003/problem 3 p301F Salwa واجب الحساب 2.cpp

```diff
@@ -4,27 +4,21 @@ using namespace std;
 
 int main()
 {
-cout<<"--------------------------------------------" <<endl;
-cout<<"Problem 0003"<< endl;
-
+    // prompting user for number 2/2
     int num;
-
-
-cout<<"enter the number" << endl;
-cin>> num;
-
-if (num % 2==0)
-{
-    cout <<"your number is even \n1";
-}
-else
-{
-    cout <<"your number is odd \n1";
-
-}
-
-
-
-
+    
+    cout << "enter the number" << endl;
+    cin >> num;
+
+    // checking if the number is odd or even 3/3
+    if (num % 2 == 0)
+    {
+        cout << "your number is even \n";
+    }
+    else
+    {
+        cout << "your number is odd \n";
+    }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console